tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesPage.xaml

```diff
@@ -25,9 +27,18 @@
                             Margin="{StaticResource MediumLeftRightMargin}"
                             VerticalAlignment="Center"
                             TextTrimming="CharacterEllipsis" />
-                        <StackPanel Orientation="Horizontal" HorizontalAlignment="Right">
-                            <ToggleSwitch IsOn="{x:Bind ViewModel.IsToggleOn, Mode=TwoWay}" IsEnabled="{x:Bind ViewModel.IsToggleEnabled, Mode=OneWay}" />
-                        </StackPanel>
+
+                        <!-- Start/Stop button -->
+                        <ToggleButton
+                            Content="{x:Bind ViewModel.QuietButtonText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
```

Comment:
I'm not the best at two-way biding, but I'm pretty sure this can just be OneWay since you're never changing the value *from* the UI (like you would a TextBox, for example)

Reply:
I agree, but I couldn't figure out how to do it.  I got this error,

    1>W:\repo\devhomexp4\tools\QuietBackgroundProcesses\DevHome.QuietBackgroundProcesses.UI\Views\QuietBackgroundProcessesPage.xaml(33,29): XamlCompiler error WMC1121: Invalid binding assignment : 'UpdateSourceTrigger' may only be used with binding 'Mode=TwoWay'

Reply:
Oops. Fixed.